docs/build/platform-build-guides/building-ios-applications.md

```diff
@@ -255,6 +255,62 @@ end
 ```
 :::
 
+### Xcode 16 Known Issues
+
+Apple introduces big changes with Xcode, which it develops every year. You can find the errors and solutions encountered for the first time after the Xcode 16 transition under this title.
+
+#### Error: `PBXGroup` attempted to initialize an object with unknown ISA `PBXFileSystemSynchronizedRootGroup`
+
+After Xcode 16 migration, when you add a new target to your project, you will get a reference error if this target is not added correctly. This error is caused by Xcode writing the reference differently in the `.pbxproj` file, which must be read to compile the project and where the project references are written.
```

Comment:
```suggestion
Following the Xcode 16 migration, adding a new target to your project may result in a reference error if the target is not configured correctly. This error occurs because Xcode now writes the reference differently in the .pbxproj file, which is essential for compiling the project and contains all project references.
```